src/features.js

```diff
@@ -30,8 +30,7 @@ export const featureDetectionPromise = Promise.resolve(supportsDynamicImportChec
       const iframe = document.createElement('iframe');
       iframe.style.display = 'none';
       document.head.appendChild(iframe);
-      // we use document.write here because eg Weixin built-in browser doesn't support setting srcdoc
-      iframe.contentWindow.document.write(`<script type=importmap nonce="${nonce}">{"imports":{"x":"data:text/javascript,"}}<${''}/script><script nonce="${nonce}">import('x').then(()=>1,()=>0).then(v=>parent._$s(v))<${''}/script>`);
+      iframe.src = URL.createObjectURL(new Blob([`<script type=importmap nonce="${nonce}">{"imports":{"x":"data:text/javascript,"}}<${''}/script><script nonce="${nonce}">import('x').then(()=>1,()=>0).then(v=>parent._$s(v))<${''}/script>`], {type : 'text/html'}))
```

Comment:
We also already have a helper for this that can be imported - https://github.com/guybedford/es-module-shims/blob/main/src/common.js#L5.

Reply:
Of course we do 🤦 
Updated here: https://github.com/guybedford/es-module-shims/pull/227/commits/84e792f4283807e0309a1fec43cae433ebd5bcea